reflex/components/component.py

```diff
@@ -675,15 +688,6 @@ def validate_valid_child(child_name):
                     f"The component `{comp_name}` only allows the components: {valid_child_list} as children. Got `{child_name}` instead."
                 )
 
-        def validate_vaild_parent(child_name, valid_parents):
```

Comment:
this function was recently added to `main`, make sure you have rebased your branch on the latest